app/models/Service.ts

```diff
@@ -27,6 +27,7 @@ export interface Service {
   categories: Category[];
   certified_at: string | null;
   certified: boolean;
+  documents: unknown[];
```

Comment:
Including `documents` is an interesting conundrum since I'm not sure that we'll use documents again, now that we dynamically generate them, but I guess we can just leave it until we make a change to the actual API itself.

Reply:
This is actually required or else the `delete currentService.documents` statement in the `postServices()` function will fail to type check. TypeScript isn't actually great for the kind of code we have there, since TypeScript's interface/object types only guarantee the _minimum_ set of properties that will exist, but provide no guarantee that you won't find other properties.

The code we have in `postServices()` is pretty bad, IMO, and we should replace it with code that constructs the service object we'll POST to the API from scratch rather than by deleting stuff off the internal state variable until it matches the shape needed by the API. I'll be happy to finally be able to do that after getting through this TypeScript migration and after the new, non-ChangeRequest API endpoints @jjfreund's been working on are available!